apm-sniffer/apm-sdk-plugin/kafka-plugin/src/main/java/org/apache/skywalking/apm/plugin/kafka/CallbackAdapterInterceptor.java

```diff
@@ -45,8 +45,8 @@ public CallbackAdapterInterceptor(CallbackCache callbackCache) {
     public void onCompletion(RecordMetadata metadata, Exception exception) {
         ContextSnapshot snapshot = callbackCache.getSnapshot();
         AbstractSpan activeSpan = ContextManager.createLocalSpan("Kafka/Producer/Callback");
-        SpanLayer.asMQ(activeSpan);
         activeSpan.setComponent(ComponentsDefine.KAFKA_PRODUCER);
+        activeSpan.setLayer(SpanLayer.MQ);
```

Comment:
Why change this code style?

Reply:
There was a PR addition submitted by someone else the other day, but I had already done it earlier, so in the merge, it was in this style

Reply:
My point is, they are nearly the same. Why change it again?

Reply:
Np.  fixed